.github/workflows/deploy-prod.yaml

```diff
@@ -6,6 +6,21 @@ on:
     - "v*.*.*"
 
 jobs:
+  verify-tag:
+    runs-on: ubuntu-20.04
+    outputs:
+        result: ${{ steps.verify-tag-id.outputs.result }}
```

Comment:
no need to output here as you are not using this as an input https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs